Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -2509,6 +2514,32 @@ bool CLuaElementDefs::SetLowLodElement(lua_State* luaVM, CClientEntity* pEntity,
     return CStaticFunctionDefinitions::SetLowLodElement(*pEntity, pLowLodEntity.value_or(nullptr));
 }
 
+bool CLuaElementDefs::SetElementOnFire(CClientEntity* entity, bool onFire)
+{
+    switch (entity->GetType())
+    {
+        case CCLIENTPED:
+        case CCLIENTPLAYER:
+        {
+            static_cast<CClientPed*>(entity)->SetOnFire(onFire);
+            return true;
+        }
+        case CCLIENTVEHICLE:
+        {
+            static_cast<CClientVehicle*>(entity)->SetOnFire(onFire);
+            return true;
+        }
+        case CCLIENTOBJECT:
+        case CCLIENTWEAPON:
+        {
+            static_cast<CClientObject*>(entity)->SetOnFire(onFire);
+            return true;
```

Comment:
What do you think about making a method available for the element class instead so you could just do `entity->SetOnFire(onFire);`?

Reply:
Good idea